packages/frontend/core/src/modules/app-tabs-header/views/tab-view-meta.tsx

```diff
@@ -0,0 +1,35 @@
+import { useI18n } from '@affine/i18n';
+import { useServiceOptional } from '@toeverything/infra';
+import { useEffect } from 'react';
+
+import { DesktopStateSynchronizer } from '../../workbench/services/desktop-state-synchronizer';
+import { ViewService } from '../../workbench/services/view';
+import type { TabIconName } from '../constants';
+
+export interface SplitViewTabMeta {
+  title?: string | null;
+  icon: TabIconName;
+}
+
+/**
+ * Used for updating the meta of a split view tab on the desktop client.
+ */
+export const ViewTabMeta = (meta: SplitViewTabMeta) => {
```

Comment:
maybe split this to `ViewTitle` and `ViewIcon`,  I feel that the concept contained in the name ViewTabMeta is a bit complicated.

Reply:
I think this may cause redundant electron api calls so it is merged into one. Also I feel like there is no obvious benefit to split it into two.